packages/field-base/src/delegate-focus-mixin.js

```diff
@@ -48,11 +48,19 @@ export const DelegateFocusMixin = dedupingMixin(
            * By default, the host element does not have tabindex attribute. Instead, `focusElement` should have it.
            * Toggling `tabindex` attribute on the host element propagates its value to `focusElement`.
            *
+           * @override
            * @protected
            */
           tabindex: {
-            type: Number,
-            value: undefined
+            type: Number
+          },
+
+          /**
+           * @protected
+           * @override
+           */
+          _lastTabIndex: {
```

Comment:
**note:** It turned out that `DelegateFocusMixin` requires `_lastTabIndex` to be `0` by default even though the `tabindex` attribute is `undefined` by default. 

If I remove it, the tests start failing. Seems that the `__forwardTabIndex` method is tied on the `0` value. It looks suspicious and it might be that there is a bug, but I decided to leave it as it is for now.

https://github.com/vaadin/web-components/pull/3627/files#diff-4923b95f38b57836c910a9c0e6b335bbb33fde76316648c81678b38e3e9898e9R216-R233